Buildings/Fluid/Humidifiers/EvaporativeCoolers/Baseclasses/DirectCalculations.mo

```diff
@@ -0,0 +1,154 @@
+within Buildings.Fluid.Humidifiers.EvaporativeCoolers.Baseclasses;
+block DirectCalculations
+  "Calculates the water vapor mass flow rate of a direct evaporative coolder"
+
+  replaceable package Medium = Modelica.Media.Interfaces.PartialMedium
+    "Medium";
+
+  parameter Modelica.Units.SI.Area padAre
+    "Area of the rigid media evaporative pad";
+
+  parameter Modelica.Units.SI.Length dep
+    "Depth of the rigid media evaporative pad";
+
+  parameter Real effCoe[11] = {0.792714, 0.958569, -0.25193, -1.03215, 0.0262659,
```

Comment:
Can the user change the `effCoe[:]`? Is it from the E+ reference? If it shall not be changed, make it to be protected.